### ISSUE TITLE
point exception (core dumped) at H20

### ISSUE THREAD

**Issue by kaixinchen2024** [MoonshotAI/Kimi-Audio#55]
root@2b4646505a80:/app/km/Kimi-Audio-master# python3 a.py
2025-04-29 17:05:41.555 | INFO     | kimia_infer.api.kimia:__init__:16 - Loading kimi-audio main model
using normal flash attention
Loading checkpoint shards: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 36/36 [00:00<00:00, 1342.31it/s]
2025-04-29 17:05:49.477 | INFO     | kimia_infer.api.prompt_manager:__init__:30 - Looking for resources in /app/km/checkpoints/Kimi-Audio-7B-Instruct
2025-04-29 17:05:49.478 | INFO     | kimia_infer.api.prompt_manager:__init__:31 - Loading whisper model
2025-04-29 17:05:51.549 | INFO     | kimia_infer.api.prompt_manager:__init__:40 - Loading text tokenizer
2025-04-29 17:05:51.734 | INFO     | kimia_infer.api.kimia:__init__:39 - Loading detokenizer
Detected CUDA files, patching ldflags
Emitting ninja build file /app/km/Kimi-Audio-master/kimia_infer/models/detokenizer/vocoder/alias_free_activation/cuda/build/build.ninja...
/usr/local/lib/python3.12/dist-packages/torch/utils/cpp_extension.py:1965: UserWarning: TORCH_CUDA_ARCH_LIST is not set, all archs for visible cards are included for compilation.
If this is not desired, please set os.environ['TORCH_CUDA_ARCH_LIST'].
  warnings.warn(
Building extension module anti_alias_activation_cuda...
Allowing ninja to set a default number of workers... (overridable by setting the environment variable MAX_JOBS=N)
[1/3] c++ -MMD -MF anti_alias_activation.o.d -DTORCH_EXTENSION_NAME=anti_alias_activation_cuda -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1011\" -isystem /usr/local/lib/python3.12/dist-packages/torch/include -isystem /usr/local/lib/python3.12/dist-packages/torch/include/torch/csrc/api/include -isystem /usr/local/lib/python3.12/dist-packages/torch/include/TH -isystem /usr/local/lib/python3.12/dist-packages/torch/include/THC -isystem /usr/local/cuda/include -isystem /usr/include/python3.12 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17 -O3 -c /app/km/Kimi-Audio-master/kimia_infer/models/detokenizer/vocoder/alias_free_activation/cuda/anti_alias_activation.cpp -o anti_alias_activation.o
[2/3] /usr/local/cuda/bin/nvcc --generate-dependencies-with-compile --dependency-output anti_alias_activation_cuda.cuda.o.d -DTORCH_EXTENSION_NAME=anti_alias_activation_cuda -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1011\" -isystem /usr/local/lib/python3.12/dist-packages/torch/include -isystem /usr/local/lib/python3.12/dist-packages/torch/include/torch/csrc/api/include -isystem /usr/local/lib/python3.12/dist-packages/torch/include/TH -isystem /usr/local/lib/python3.12/dist-packages/torch/include/THC -isystem /usr/local/cuda/include -isystem /usr/include/python3.12 -D_GLIBCXX_USE_CXX11_ABI=0 -D__CUDA_NO_HALF_OPERATORS__ -D__CUDA_NO_HALF_CONVERSIONS__ -D__CUDA_NO_BFLOAT16_CONVERSIONS__ -D__CUDA_NO_HALF2_OPERATORS__ --expt-relaxed-constexpr -gencode=arch=compute_90,code=compute_90 -gencode=arch=compute_90,code=sm_90 --compiler-options '-fPIC' -O3 -gencode arch=compute_70,code=sm_70 --use_fast_math -U__CUDA_NO_HALF_OPERATORS__ -U__CUDA_NO_HALF_CONVERSIONS__ --expt-relaxed-constexpr --expt-extended-lambda -gencode arch=compute_80,code=sm_80 -std=c++17 -c /app/km/Kimi-Audio-master/kimia_infer/models/detokenizer/vocoder/alias_free_activation/cuda/anti_alias_activation_cuda.cu -o anti_alias_activation_cuda.cuda.o
[3/3] c++ anti_alias_activation.o anti_alias_activation_cuda.cuda.o -shared -L/usr/local/lib/python3.12/dist-packages/torch/lib -lc10 -lc10_cuda -ltorch_cpu -ltorch_cuda -ltorch -ltorch_python -L/usr/local/cuda/lib64 -lcudart -o anti_alias_activation_cuda.so
Loading extension module anti_alias_activation_cuda...
/usr/local/lib/python3.12/dist-packages/torch/nn/utils/weight_norm.py:134: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
Loading '/app/km/checkpoints/Kimi-Audio-7B-Instruct/vocoder/model.pt'
Complete.
using rope base theta = 10000.0, interpolation factor = 1.0
Currently using bfloat16 for PrefixFlowMatchingDetokenizer
Generating tokens:   0%|                                                                                                                               | 1/7357 [00:00<23:59,  5.11it/s]Floating point exception (core dumped)

**Comment by codingDuan** [comment]
+1

**Comment by muverystrong** [comment]
a100没问题，h20可以采用下边的方案：
pip install nvidia-cublas-cu12==12.4.5.8
同时升级pytorch, pytorchaudio到2.6.1
